Header header
bool flag_take_off
bool flag_landing
bool flag_go_home
bool flag_arrive_target_area
bool flag_search_task_end
bool flag_auto_landing_end
